lib/identity_config.rb

```diff
@@ -181,6 +181,7 @@ def self.build_store(config_map)
     config.add(:irs_attempt_api_enabled, type: :boolean)
     config.add(:irs_attempt_api_event_ttl_seconds, type: :integer)
     config.add(:irs_attempt_api_public_key)
+    config.add(:kantara_currently_required, type: :boolean)
```

Comment:
The kantara certification covers a bunch of different things, WDYT about makign this more specific to the 2FA + phone restriction? `kantara_2fa_phone_restricted`

Reply:
This makes sense. Thanks!